docker-base/Dockerfile.alpine

```diff
@@ -48,7 +48,7 @@ RUN apk add --no-cache \
         "linux/arm/v7") GIT_LFS_ARCH=arm ;; \
     esac && \
     curl -L -s --output git-lfs.tar.gz "https://github.com/git-lfs/git-lfs/releases/download/v${GIT_LFS_VERSION}/git-lfs-linux-${GIT_LFS_ARCH}-v${GIT_LFS_VERSION}.tar.gz" && \
-    tar -xf git-lfs.tar.gz && \
+    tar --strip-components=1 -xf git-lfs.tar.gz && \
```

Comment:
Why is this flag `--strip-components` added?

Reply:
I see the commit here https://github.com/runatlantis/atlantis/pull/2889/commits/f802e9d0df9c5d9d7c0e0ff51b2ff3ba43e3905a

Reply:
In `git-lfs` `3.1.2` archive, the `git-lfs` object was located at the root level, but in `3.3.0` everything is located in the `git-lfs-3.3.0/` path, so at extraction, everything is put into the `git-lfs-3.3.0` folder. We either need to step into that folder, or remove that component at extraction, which the command solves.

`tar -tvf git-lfs.tar.gz`:
`-rwxr-xr-x  0 runner docker 10739712 Feb 16  2022 git-lfs`
vs
`-rwxr-xr-x  0 runner docker 10993664 Nov 30 18:26 git-lfs-3.3.0/git-lfs`

Reply:
Great explanation. Thank you.